docs/avdanos/contributing/design/OXYDE/main-style.md

```diff
@@ -0,0 +1,121 @@
+---
+sidebar_position: 2
+title: Main style
+description: The main style of our design.
+---
+
+:::caution
+Reminder: OXYDE stills in Beta phase.
+:::
+
+# Fonts
+
+When designing, fonts are an important point. AvdanOS font for User Interface is **Ubuntu**.
+![OXYDE Fonts](Docsite/static/img/Fonts.png)

```

Comment:
For absolutely no reason there is a compiling error:
![image](https://user-images.githubusercontent.com/87482502/216966617-cfd1ee3a-1dfa-428c-b859-d982b7d4ac79.png)

Fix:
Just Remove `Docsite/static` In All Of the places

```suggestion
![OXYDE Fonts](/img/Fonts.png)

```